calculated_bot.py

```diff
@@ -320,8 +362,45 @@ def chunks(l, n):
     args = ctx.message.content.split(" ")
 
     if len(args) < 2:
-        await bot.send_message(ctx.message.channel,
-                               f"Not enough arguments! The proper form of this command is: `{BOT_PREFIX}ranks <id>`")
+        PastUsers = open("UserIDs.txt", "r+")
+        NotLinked = True
+        for line in PastUsers:
+            check = line
+            check = check.strip()
+            check = check.split(":")
```

Comment:
You can chain commands:

```suggestion
            check = line.strip().split(":")
```